pkgs/by-name/cu/cutentr/package.nix

```
@@ -0,0 +1,77 @@
+{
+  stdenv,
+  libsForQt5,
+  fetchFromGitLab,
+  makeDesktopItem,
+  fetchurl,
+  lib,
+  copyDesktopItems,
+}:
+
+let
+  version = "0.3.3";
+
+  icon = fetchurl {
+    url = "https://gitlab.com/BoltsJ/cuteNTR/-/raw/${version}/setup/gui/com.gitlab.BoltsJ.cuteNTR.svg";
+    hash = "sha256-bnSNJh13E2U11b+qW0SjfvZQ/VQi5Dbuz65g3svTgWo=";
+  };
+in
+
+stdenv.mkDerivation {
+  pname = "cutentr";
+  inherit version;
+
+  src = fetchFromGitLab {
+    owner = "BoltsJ";
+    repo = "cuteNTR";
+    tag = version;
+    hash = "sha256-KfnC9R38qSMhQDeaMBWm1HoO3Wzs5kyfPFwdMZCWw4E=";
+  };
+
+  desktopItems = lib.singleton (makeDesktopItem {
+    name = "cuteNTR";
+    desktopName = "cuteNTR";
+    icon = "cutentr";
+    exec = "cutentr";
+    categories = [ "Game" ];
+  });
+
+  nativeBuildInputs =
+    with libsForQt5.qt5;
+    [
+      wrapQtAppsHook
+    ]
+    ++ [
+      copyDesktopItems
+    ];
```

nit: would personally format like this (doesn't really matter)
```suggestion
  nativeBuildInputs = [
    libsForQt5.wrapQtAppsHook
    copyDesktopItems
  ];
```